compiler/tests-common/tests/org/jetbrains/kotlin/codegen/AbstractCheckLocalVariablesTableTest.kt

```diff
@@ -46,6 +46,7 @@ abstract class AbstractCheckLocalVariablesTableTest : CodegenTestCase() {
             assertNotNull("Couldn't find class file for pattern $classFileRegex in: $pathsString", outputFile)
 
             val actualLocalVariables = readLocalVariable(ClassReader(outputFile.asByteArray()), methodName)
+            checkLocalVariableTypes(ClassReader(outputFile.asByteArray()), methodName, actualLocalVariables)
```

Comment:
Could we reuse class reader from previous line?

Reply:
Yes, good idea. Done.